uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemImplProviderSSHTest.java

```diff
@@ -118,7 +120,7 @@ public String getName() {
         //Push clone back to origin
         provider.getFileSystem(URI.create("git://repo-clone?push=ssh://admin@localhost:" + gitSSHPort + "/repo"));
 
-        verify(hook).execute("repo");
+        verify(hook).execute(any());
```

Comment:
could we please also verify it was called with expected `FileSystemHookExecutionContext` ?

Reply:
done